.github/workflows/pr-open.yml

```
@@ -1,50 +1,115 @@
-      NEXT_PUBLIC_TINA_BRANCH: ${{ github.head_ref }}
-      NEXT_PUBLIC_TINA_CLIENT_ID: ${{ secrets.NEXT_PUBLIC_TINA_CLIENT_ID }}
-      TINA_TOKEN: ${{ secrets.TINA_TOKEN }}
-      RECAPTCHA_SECRETKEY: 'test'
-      RESEND_API_KEY: 're_test'
-      EMAIL_FROM: 'test'
-      NEXT_PUBLIC_RECAPTCHA_SITEKEY: 'test'
-      NEXT_PUBLIC_URL: 'https://www.nerdfish.be'
-      GROQ_API_KEY: 'test'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)